src/core/messenger/background.ts

```diff
@@ -229,6 +229,60 @@ const setupBackground = async (
 				video.style.height = '100%';
 				video.style.transform = 'translate(-50%, -50%)';
 				background.appendChild(video);
+
+				if (!window.guardian.config.switches.sentinelLogger) return;
+
+				const getCreativeId = () => {
+					const slots = googletag.pubads().getSlots();
+
+					for (const slot of slots) {
+						const creativeTemplateId =
+							slot.getResponseInformation()?.creativeTemplateId;
+						if (creativeTemplateId === 11885667) {
+							return slot.getResponseInformation()?.creativeId;
+						}
+					}
+					return null;
+				};
+
+				const sendVideoProgress = () => {
+					const endpoint = window.guardian.config.page.isDev
+						? '//logs.code.dev-guardianapis.com/log'
+						: '//logs.guardianapis.com/log';
```

Comment:
Another option for the data collection is to create an `EventTimer.mark()` so it is sent along with the commercial metrics to the data-lake.

https://github.com/guardian/commercial/blob/f935c3dd1e56f142912dc3399d0e2ee7e8208bba/src/core/event-timer.ts#L215

The `track-scroll-depth` module uses this approach:

https://github.com/guardian/commercial/blob/main/src/core/track-scroll-depth.ts#L15-L16

Was there any particular reason why we have opted to use the fastly endpoint here?

Reply:
No strong reason! It's the same endpoint we use for the `amiused` function so the CODE env was empty and easy to use for testing. Happy to switch to using the same endpoint as commercial metrics though if that's preferred. Does using `EventTimer.mark()` allow us to send two values together (the video progress and interscroller creativeId) or would we need to add them to metrics as separate properties?

Reply:
I've made the change to add the data to the commercial metrics payload instead